Readme.md

```diff
@@ -119,6 +120,14 @@ If true, depending on the [lazyStart](#WorkerNodesOptions+lazyStart) option, it
 ### options.lazyStart : <code>Boolean</code>
 Whether should start a new worker only if all the others are busy.
 
+**Kind**: instance property of [<code>WorkerNodesOptions</code>](#WorkerNodesOptions)  
+**Default**: <code>false</code>  
+<a name="WorkerNodesOptions+asyncWorkerInitialization"></a>
+
+### options.asyncWorkerInitialization : <code>Boolean</code>
+Enables async initialization of worker.
+To start handling task over worker, need to init `sendWorkerMessage('ready')` function when it fully initialized.
```

Comment:
Maybe `need to invoke`? Also, it may be useful to link to `async-initialization.spec.js`.

Reply:
done !